webapp/graphite/render/datalib.py

```diff
@@ -150,32 +148,20 @@ def copy(self, name=None, start=None, end=None, step=None, values=None, consolid
     )
 
 
+# Data retrieval API
 @logtime(custom_msg=True)
-def _fetchData(pathExpr, startTime, endTime, now, requestContext, seriesList, msg_setter=None):
+def fetchData(requestContext, pathExpr, msg_setter=None):
```

Comment:
same here, `msg_setter` could be `None` and then the next line would throw an exception

Reply:
it needs to have a default value to avoid codacy issues, it will never actually be null since it's passed by the decorator